app/assets/styles/partials/base/_print.scss

```diff
@@ -1,3 +1,5 @@
+/* stylelint-disable */
```

Comment:
Is this disable because of the '!important' as below we use *stylelint-disable declaration-no-important*

Reply:
the print stylesheet would require quite a refactor to pass lint so I just disabled it completely